### ISSUE TITLE
Update details on HZB

### ISSUE THREAD

**Issue by agbeltran** [icatproject/icatproject.github.io#29]
We have a page describing HZB use of ICAT (https://icatproject.org/collaboration/facilities/hzb/), @RKrahl do you think it needs updating? It would be good to include some links to HZB and perhaps also the catalogue.

**Comment by RKrahl** [comment]
Yes, that page is a little dated.  In particular, BER II is history meanwhile.  We could also add a link to the [HZB ICAT](https://data.helmholtz-berlin.de/).

Unfortunately, we didn't advance as fast as I would have wanted since then.  I would be much more enthusiastic about updating the text if I had some shiny new results to put there, but this is not the case.  In any case, I would need to get clearance for a new text version from our communication department.

**Comment by agbeltran** [comment]
Thank you @RKrahl - sure, let us know when you are ready to update the page. We can add the HZB link and the catalogue link as a minimum.
